packages/components/autocomplete/src/Autocomplete.tsx

```diff
@@ -75,6 +75,11 @@ export interface AutocompleteProps<ItemType>
    * @default false
    */
   clearAfterSelect?: boolean;
+  /**
+   * If this is set to `true` the dropdown menu will stay open after selecting an item
```

Comment:
```suggestion
   * If this is set to `false` the dropdown menu will stay open after selecting an item
```

I think my suggestion got messed up.